x-pack/plugins/security_solution/common/endpoint/schema/policy.ts

```diff
@@ -19,3 +19,21 @@ export const GetAgentPolicySummaryRequestSchema = {
     policy_id: schema.nullable(schema.string()),
   }),
 };
+
+const ListWithKuerySchema = schema.object({
```

Comment:
would be good to add tests for theses. I have a suspicion that these default values might not work since they're wrapped in a maybe.

Reply:
example schema tests: https://github.com/elastic/kibana/blob/main/x-pack/plugins/security_solution/common/endpoint/schema/metadata.test.ts.